scico/test/functional/test_loss.py

```diff
@@ -208,3 +208,12 @@ def test_prox(self, loss_tuple):
 
         pf = prox_test((1 + 1j) * snp.zeros(self.v.shape), L, L.prox, 0.0)  # complex zero v
         pf = prox_test((1 + 1j) * snp.zeros(self.v.shape), L, L.prox, 1.0)  # complex zero v
+
+
+def test_cubic_root():
+    N = 10000
+    p, _ = uniform(shape=(N,), dtype=snp.float32, minval=-10.0, maxval=10.0)
+    q, _ = uniform(shape=(N,), dtype=snp.float32, minval=-10.0, maxval=10.0)
```

Comment:
jax gotcha: I think p and q will be equal if you make them this way. Need to get the key from the first call and pass it to the second.

Reply:
Indeed they were. Fixed.